linux/build.gradle

```diff
@@ -54,4 +54,18 @@ def getInputDir() {
 
 def getLocalBuildStatus() {
 	return hasProperty("INPUT_DIR") ? "true" : "false"
+}
+
+def getContainerRegistry() {
```

Comment:
Maybe this is more getUsesContainerRegistry - I don't think it returns the actual registry

Reply:
It does return the container registry if it is specified, otherwise it returns an empty string that reference to the default container registry `docker.io` (this is the default in docker).

Reply:
OK, but then there is a caller below that's using it in an if(<predicate>) clause.  Are you relying on a non empty string == "True" in groovy for that check (~Line 127)?

Reply:
Yes, I am relying on a non empty string == "True"